### ISSUE TITLE
[a11y] Checkboxes should be blue 600

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#16394]
For sufficient color contrast checkboxes (and similar items) should be blue600. They currently blue500.

**Comment by jonahwilliams** [comment]
@goderbauer I'm assuming for similar items this contrast should apply to Checkboxes, RadoButtons, Switches, and Slider?

**Comment by goderbauer** [comment]
Correct.
